common/src/comments.rs

```diff
@@ -37,6 +37,8 @@ pub trait Comments {
     fn has_trailing(&self, pos: BytePos) -> bool;
     fn move_trailing(&self, from: BytePos, to: BytePos);
     fn take_trailing(&self, pos: BytePos) -> Option<Vec<Comment>>;
+
+    fn add_pure_comment(&self, pos: BytePos);
```

Comment:
Can you add a default implementation using `take_leading` and `add_leading`?
In this way the pr would be non-breaking change.

Reply:
Good idea. I'm still a Rust noob and didn't know this was possible. Thanks!